src/app/teacher/archive-projects-button/archive-projects-button.component.html

```diff
@@ -0,0 +1,24 @@
+<button
+  mat-icon-button
+  *ngIf="!showArchived"
+  (click)="archiveSelectedProjects(true)"
+  aria-label="Archive selected units"
+  i18n-aria-label
+  matTooltip="Archive"
+  matTooltipPosition="above"
+  i18n-matTooltip
```

Comment:
You don't need aria-label when you have mat-tooltip?
```suggestion
  matTooltip="Archive selected units"
  matTooltipPosition="above"
  i18n-matTooltip
```